src/Lykke.Job.NeoClaimTransactionsExecutor/Startup.cs

```diff
@@ -0,0 +1,70 @@
+﻿using JetBrains.Annotations;
+using Lykke.Job.NeoClaimTransactionsExecutor.Settings;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.Extensions.DependencyInjection;
+using System;
+using Lykke.Logs.Loggers.LykkeSlack;
+using Lykke.Sdk;
+using Microsoft.Extensions.Logging;
+
+namespace Lykke.Job.NeoClaimTransactionsExecutor
+{
+    [UsedImplicitly]
+    public class Startup
+    {
+        private readonly LykkeSwaggerOptions _swaggerOptions = new LykkeSwaggerOptions
+        {
+            ApiTitle = "NeoClaimTransactionsExecutorJob API",
+            ApiVersion = "v1"
+        };
+
+        [UsedImplicitly]
+        public IServiceProvider ConfigureServices(IServiceCollection services)
+        {
+            return services.BuildServiceProvider<AppSettings>(options =>
+            {
+                options.SwaggerOptions = _swaggerOptions;
+
+                options.Logs = logs =>
+                {
+                    logs.AzureTableName = "NeoClaimTransactionsExecutorJobLog";
+                    logs.AzureTableConnectionStringResolver = settings => settings.NeoClaimTransactionsExecutorJob.Db.LogsConnString;
+
+                    // TODO: You could add extended logging configuration here:
+
+                    logs.Extended = p =>
+                    {
+                        p.AddAdditionalSlackChannel("CommonBlockChainIntegration");
```

Comment:
Add these lines for both additional channels https://github.com/LykkeCity/Lykke.Job.BlockchainCashoutProcessor/blob/master/src/Lykke.Job.BlockchainCashoutProcessor/Startup.cs#L77-L78